packages/cli/src/constructs/dashboard.ts

```diff
@@ -0,0 +1,212 @@
+   * Show or hide header and description on the dashboard. Default: true.
+   */
+  showHeader?: boolean
+  /**
+   * Custom CSS to be applied to the dashboard.
+   */
+  customCSS?: Content|Entrypoint
+  /**
+   * Determines if the dashboard is public or private. Default: false.
+   * @description only paid accounts can enable this feature.
+   */
+  isPrivate?: boolean
+  /**
+   * Show or hide the P95 stats on the dashboard. Default: true.
+   */
+  showP95?: boolean
+  /**
+   * Show or hide the P99 stats on the dashboard. Default: true.
+   */
+  showP99?: boolean
+}
+
+/**
+ * Creates a Dashboard
+ *
+ * @remarks
+ *
+ * This class make use of the Dashboard endpoints.
+ */
+export class Dashboard extends Construct {
+  tags?: Array<string>
+  customUrl?: string
+  customDomain?: string
+  logo?: string
+  favicon?: string
+  link?: string
+  header?: string
+  description?: string
+  width?: 'FULL'|'960PX'
+  refreshRate?: 60|300|600
+  paginate?: boolean
+  paginationRate?: 30|60|300
+  checksPerPage?: number
+  useTagsAndOperator?: boolean
+  hideTags?: boolean
+  enableIncidents?: boolean
+  expandChecks?: boolean
+  showHeader?: boolean
+  customCSS?: string
+  isPrivate?: boolean
+  showP95?: boolean
+  showP99?: boolean
+
+  static readonly __checklyType = 'dashboard'
+
+  /**
+   * Constructs the Dashboard instance
+   *
+   * @param logicalId unique project-scoped resource name identification
+   * @param props dashboard configuration properties
+   */
+  constructor (logicalId: string, props: DashboardProps) {
+    super(Dashboard.__checklyType, logicalId)
+    this.tags = props.tags
+    this.customUrl = props.customUrl
+    this.customDomain = props.customDomain
+    this.logo = props.logo
+    this.favicon = props.favicon
+    this.link = props.link
+    this.header = props.header
+    this.description = props.description
+    this.width = props.width
+    this.refreshRate = props.refreshRate
+    this.paginate = props.paginate
+    this.paginationRate = props.paginationRate
+    this.checksPerPage = props.checksPerPage
+    this.useTagsAndOperator = props.useTagsAndOperator
+    this.hideTags = props.hideTags
+    this.enableIncidents = props.enableIncidents
+    this.expandChecks = props.expandChecks
+    this.showHeader = props.showHeader
+    this.isPrivate = props.isPrivate
+    this.showP95 = props.showP95
+    this.showP99 = props.showP99
+
+    if (!props.customUrl && !props.customDomain) {
+      throw new Error('Either a "customUrl" or "customDomain" must be specified.')
+    }
+
+    if (props.customCSS) {
+      if ('entrypoint' in props.customCSS) {
+        if (!fs.existsSync(props.customCSS.entrypoint)) {
+          throw new Error(`Unrecognized CSS code for 'customCSS' property in dashboard '${logicalId}'.`)
+        }
+        this.customCSS = String(fs.readFileSync(props.customCSS.entrypoint))
+      } else if ('content' in props.customCSS) {
+        this.customCSS = props.customCSS.content
+      }
+    }
```

Comment:
Could you check how we do this in `browser-check.ts` with relative paths and implement it the same way so there is no difference?

Reply:
@umutuzgur , I checked it but then I saw the setup/tearDown script are different than the browser checks. So, I followed the same pattern for scripts. I'll use the browser checks files way now.

Reply:
You must have the older version. `api-check.ts` does also something similar atm